Formula/freeimage.rb

```diff
@@ -25,6 +25,10 @@ class Freeimage < Formula
   end
 
   def install
+    if Hardware::CPU.arm?
```

Comment:
```suggestion
    # Temporary workaround for ARM. Upstream tracking issue:
    # https://sourceforge.net/p/freeimage/discussion/36111/thread/cc4cd71c6e/
    if Hardware::CPU.arm?
```

Reply:
Strictly speaking, this condition isn't needed since the define doesn't do anything on x86.

Reply:
One more thing... I just created #68354 that adds a head URL to this formula. If you make changes to this commit, could you guard these changes with build.stable?

This is the change I'm using on my machine: https://github.com/andysan/homebrew-core/commit/a96574c9ba2cd36d1fe440a43184654c9e9d3ee5